### DISCUSSION TITLE
Unable to execute $ git push -u origin main, get error 403 forbidden

### DISCUSSION THREAD

**Discussion by Billjay65** [community/community#35478] [Other Feature Feedback, Questions, & Ideas]
Hi, 
Please I have tried to push my project to github on windows but I have been unsuccessful. I wrote to github but now I can't create a follow up ticket. When I try the command: $ git push -u origin main  , I am prompted for my username and password. I then enter my github username and my git token and I get the error: 

$ git push origin main
remote: Permission to Billjay65/humanr.git denied to Billjay65.
fatal: unable to access 'https://github.com/Billjay65/humanr.git/': The requested URL returned error: 403

I have tried to view windows credential manager but I dont't see any passwords or tokens.
Please what is the problem. This has persisted for a month now. I am exhausted trying to find a solution.
![windows-credential](https://user-images.githubusercontent.com/95188150/194653021-a813c97c-58fe-424e-aff5-6ccb77d2bba3.PNG)

**Comment by doguedogue** [comment]
try with ssh maybe your github needs that kind of security, sometimes password isn't enough to push your repository

https://docs.github.com/en/authentication/connecting-to-github-with-ssh

**Comment by airtower-luna** [comment]
> I am prompted for my username and password. I then enter my github username and my git token and I get the error:

That is the right approach in principle. Does your token have the right scopes? You need **repo** to be able to push in general, and **workflow** if you use Actions (to be able to make pushes that modify workflow files).

Also check that the token hasn't expired.
